src/components/settings/SettingsFooter.tsx

```diff
@@ -42,23 +31,6 @@ export const SettingsFooter: FC<ISettingsFooter> = ({
       >
         <div className="flex items-center gap-1">
           <span aria-label="app-version">Gitify {appVersion}</span>
-          <span className="pb-1">
-            {isUpdateAvailable ? (
-              <span title="New version available">
-                <AlertFillIcon
-                  size={Size.XSMALL}
-                  className={IconColor.YELLOW}
-                />
-              </span>
-            ) : (
-              <span title="You are using the latest version">
-                <CheckCircleFillIcon
-                  size={Size.XSMALL}
-                  className={IconColor.GREEN}
-                />
-              </span>
-            )}
-          </span>
```

Comment:
Thanks 🙏 Didn't like it either 👀

Reply:
Yeah.... That didn't turn out as I'd hoped 😅